ui/app/store/actions.js

```diff
@@ -1217,8 +1217,6 @@ export function showAccountDetail (address) {
       if (err) {
         return dispatch(displayWarning(err.message))
       }
-      // TODO: Use correct `origin` here
-      background.handleNewAccountSelected(window.origin, address)
```

Comment:
Yup, my original approach was bad.

Reply:
It made more sense in the before-time, when account selection was per-domain, since only the UI knows what the active tab dapp is.